doc/query-format/comparison-operators.md

```diff
@@ -9,7 +9,7 @@ The `$eq` operator behaves like the [$eq MongoDB equivalent](https://docs.mongod
 
 @signature `{ $ne: <value> }`
 
-The `$ne` operator behaves like the [$eq MongoDB equivalent](https://docs.mongodb.com/manual/reference/operator/query/ne/)
+The `$ne` operator behaves like the [$ne MongoDB equivalent](https://docs.mongodb.com/manual/reference/operator/query/ne/)
```

Comment:
It would be nice to give an example of all of these.

Reply:
Added to #16